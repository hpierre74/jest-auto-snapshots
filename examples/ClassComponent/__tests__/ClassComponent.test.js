import snap from 'jest-auto-snapshots';
import ClassComponent from '../ClassComponent';

snap(ClassComponent, '../ClassComponent.jsx');
