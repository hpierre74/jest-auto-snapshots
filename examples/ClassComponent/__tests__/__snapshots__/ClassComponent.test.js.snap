// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest-auto-snapshots > ClassComponent Matches snapshot when boolean prop "sayHello" is set to: "false" 1`] = `
<div>
  <span>
    Goodbye
  </span>
</div>
`;

exports[`jest-auto-snapshots > ClassComponent Matches snapshot when passed only required props 1`] = `
<div>
  <span>
    Hello
  </span>
</div>
`;
